File: dezoito/dezoito.js
let word = prompt("Digite uma palavra:");
let result = "";

for (let char of word) {
  let asciiValue = char.charCodeAt(0);
  if (asciiValue >= 65 && asciiValue <= 90) {
    asciiValue -= 32;
  }
  result += String.fromCharCode(asciiValue);
}

console.log(`A palavra '${word}' com as letras maiúsculas convertidas para minúsculas é: ${result}`);

File: dezoito.py
palavra = input("Digite uma palavra: ")
resultado = ""

for char in palavra:
    val = ord(char)
    if 65 <= val <= 90:
        val -= 32
    resultado += chr(val)

print(f"A palavra '{palavra}' com as letras maiúsculas convertidas para minúsculas é: {resultado}")